src/DurableEntities/StartDateChangedEventHandler.cs

```diff
@@ -0,0 +1,29 @@
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.DurableTask;
+using Microsoft.Extensions.Logging;
+using SFA.DAS.Apprenticeships.Types;
+using SFA.DAS.NServiceBus.AzureFunction.Attributes;
+using System.Text.Json;
+using System.Threading.Tasks;
+using ILogger = Microsoft.Extensions.Logging.ILogger;
+
+namespace SFA.DAS.Funding.ApprenticeshipEarnings.DurableEntities;
+
+public class StartDateChangedEventHandler
```

Comment:
I know the story AC's assume validation is carried out at an early stage, but is it worth having some logic in the system to prevent illegal start date changes?